scripts/dts/extract_dts_includes.py

```diff
@@ -584,15 +584,24 @@ def yaml_traverse_inherited(node):
     :return: node
     """
 
+    if 'node_type' not in node.keys():
```

Comment:
I don't know what `node` is, but if it's a dict, why use the `in` operator in the return of `keys`, instead of just using it in the `node` object directly?  Calling the `keys` method will allocate a new tuple with all the keys in the dict, and the `in` operator will then perform a linear search in that tuple; whereas, if using the `in` operator with `node` directly, a hash table lookup will be performed.

Reply:
node is a dictionary. The .keys() thing is just taken from the original file. I will change.